src/app/core/rpc/rpc.service.ts

```diff
@@ -48,7 +49,7 @@ export class RpcService implements OnDestroy {
     private _http: HttpClient,
     private _ipc: IpcService
   ) {
-    this.isElectron = window.electron;
+    this.isElectron = false; // window.electron
```

Comment:
Currently we're forcing electron over the old http request.

Reply:
The reason we want it to go via electron is for security.. The transaction confirmation dialog will send supposed to pop up from the electron side, which currently isn't implemented

Reply:
I know, we can do the same with this solution :) I just added the comment to keep track of this small hack.